designs/administrator/4.6/column-management/index.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Column Management
+
+## Behavior
+Column management will live in the toolbar as a column icon on the Pods page on desktop. It will be hidden on mobile. Clicking on the icon will open a modal that lists all column names with checkboxes.
+![column-management-toolbar](img/column-management-toolbar.png)
+If the user is looking at “all projects”, then the checkbox next to “name” as well as “namespace”  in the modal will be shown as checked and disabled. Hovering over it will bring up a tooltip that explains that the columns may not be edited.
```

Comment:
was there any discussion around having the tool tip on the label and the checkbox to make it a bit more available?

Reply:
We didn't discuss that but that'd probably be an easy and worthy addition @matthewcarleton !